### PR TITLE
Added page about command line usage

### PR DESCRIPTION
**Pull request by pgaskin**

### CODE REVIEW

File: content/doc/usage/command-line.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+date: "2017-01-01T16:00:00+02:00"
+title: "Usage: Command Line"
+slug: "command-line"
+weight: 10
+toc: true
+draft: false
+menu:
+  sidebar:
+    parent: "usage"
+    name: "Backup and Restore"
```
Comment (lunny) [inline content/doc/usage/command-line.md]: not corrected

**Review by lunny — CHANGES_REQUESTED** [review]

File: content/doc/usage/command-line.md (hunk 1 above)
Comment (pgaskin) [reply content/doc/usage/command-line.md]: oops. Thanks for pointing that out. It's quite late here :sleepy:

Comment (pgaskin) [reply content/doc/usage/command-line.md]: Fixed

Comment (bkcsoft) [reply content/doc/usage/command-line.md]: `name` is still incorrect no?

Comment (bkcsoft) [reply content/doc/usage/command-line.md]: nvm, GitHub being stupid only showing me a partial update 🙄

**Comment by bkcsoft** [comment]
LGTM

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
LGTM

**Comment by pgaskin** [comment]
This does not seem to be rendering properly. The lists are messed up. Anyone know what's going on? This does not happen when I use other markdown editors.

**Pull request merged**